pkg/certificate/certificate.go

```diff
@@ -62,3 +64,37 @@ func CAKeyPair(config *configupload.Configuration) (*rsa.PrivateKey, *x509.Certi
 
 	return rsaKey, certs[0], nil
 }
+
+func NewSignedWebhookCert(name string, namespace string, caKey crypto.Signer, caCert *x509.Certificate) (map[string]string, error) {
+	serviceCommonName := fmt.Sprintf("%s.%s.svc", name, namespace)
+	serviceFQDNCommonName := fmt.Sprintf("%s.cluster.local.", serviceCommonName)
```

Comment:
Shouldn't we use FQDN from the KubeOneCluster object instead of defaulting to `.cluster.local.`?

Reply:
yeah (it was originally like that).